types/opentok/index.d.ts

```diff
@@ -196,10 +196,16 @@ declare module 'opentok' {
       layoutClassList: string[];
       videoType: 'camera' | 'screen' | 'custom';
     }
+
+    export interface Env {
```

Comment:
Is the expectation that a user will extend this as a class? If not then this should just be a type instead of an interface

Reply:
It shouldn't be extended by our users. This repo has the `interface-over-type-literal` rule enabled so we need to use interfaces, though.

Reply:
Ahh this is ok then